browser_patches/webkit/archive.sh

```diff
@@ -75,11 +75,15 @@ createZipForLinux() {
     # Copy libvpx.so.5 as Ubuntu 20.04 comes with libvpx.so.6
     ldd WebKitBuild/WPE/Release/bin/MiniBrowser | grep -o '[^ ]*\/libvpx.so.5[^ ]*' | xargs cp -t $tmpdir
     # Copy some wayland libraries required for Web Process t
-    cp -d -t $tmpdir WebKitBuild/WPE/DependenciesWPE/Root/lib/libva\-*
+    if ls WebKitBuild/WPE/DependenciesWPE/Root/lib/libva\-* 2>&1 >/dev/null; then
```

Comment:
why is this condition added? Does it happen now that there's nothing to copy?

Reply:
I tried to adapt the script so it works both with this minimal jhbuild moduleset and with the default one (which builds many other libraries). When using this minimal moduleset, the if condition will be false.